kolibri/plugins/learn/assets/src/views/content-unavailable-page/index.vue

```diff
@@ -20,12 +20,11 @@
       adminLink:
         'Download content channels from the <a href="/management/device#/content">Content Management</a> page',
       notAdmin:
-        'You need to sign in as the Device Owner to manage content. (This is the account originally created in the Setup Wizard.)',
+        'You need to sign in as a Super User to manage content. (This is the account originally created in device onboarding.)',
```

Comment:
Would prefer not to include the super-user changes in this PR. There are many other similar changes necessary, and I'd prefer to do them all together.

Reply:
Righto, can revert.